stingray/tests/test_powerspectrum.py

```diff
@@ -287,6 +285,14 @@ def test_classical_significances_trial_correction(self):
                                           trial_correction=True)
         assert np.size(pval) == 0
 
+
+    def test_classical_significances_with_logbinned_psd(self):
+        ps = Powerspectrum(lc=self.lc, norm="leahy")
+        ps_log = ps.rebin_log()
+        pval = ps_log.classical_significances(threshold=1.1, trial_correction=False)
+
+        assert len(pval[0]) == len(ps_log.power)
```

Comment:
Do we have a way to also test that the values are in the right ballpark? Otherwise, looks good to me!